zoo-css-challenge/style.css

```diff
@@ -1,5 +1,6 @@
 * {
   font-family: sans-serif;
+  line-height: 2; /* is there a better way to do this? */
```

Comment:
This is the correct way to do this when we want to change the spacing between lines of text of the same paragraph 👍